### PR TITLE
Updated to use facebook/webdriver 1.0

### PR DESCRIPTION
**Pull request by DavertMik**

### CODE REVIEW

**Comment by javigomez** [comment]
closes #2120

**Comment by DavertMik** [comment]
they have changed naming of every class. This may look like a BC break :(

**Comment by javigomez** [comment]
ouch!

Here is the compare view: 

https://github.com/facebook/php-webdriver/compare/v0.5...1.0.0

**Comment by DavertMik** [comment]
@zbateson could you look into it? I implemented a bit more than just updating webdriver. Actually I added few new methods to WebDriver and PhpBrowser to make them simpler to use in helpers

File: composer.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
         "ext-json": "*",
         "ext-mbstring": "*",
         "phpunit/phpunit": "~4.7.0",
-        "facebook/webdriver": "~0.4|~0.5",
+        "facebook/webdriver": "~0.4|~0.5|~1.0",
         "guzzlehttp/guzzle": ">=4.0|<7.0",
```
Comment (DavertMik) [inline composer.json]: oh, probably this should be changed to `~1.0` only

File: composer.json [hunk 2]
```diff
@@ -19,7 +19,7 @@
         "ext-json": "*",
         "ext-mbstring": "*",
         "phpunit/phpunit": "~4.7.0",
-        "facebook/webdriver": "~0.4|~0.5",
+        "facebook/webdriver": "~0.4|~0.5|~1.0",
```
Comment (DavertMik) [inline composer.json]: oh, probably this should be changed to `~1.0` only

**Pull request merged**